### ISSUE TITLE
SBUS inverter

### ISSUE THREAD

**Issue by mtzm** [zendes/SBUS#1]
Hi,
the SBUS converter i made according to SBUS_inverter.png doesn't work for me.
I use this one here:
![single_inverter](https://cloud.githubusercontent.com/assets/14362655/9977198/31542d44-5eff-11e5-847e-44ffc17e940a.jpeg)

**Comment by red-green** [comment]
Yes, it seems that the supplied picture is attenuated to ground. Your schematic should be included instead.

**Comment by zendes** [comment]
I've applied red-green's commits to the repo, so it now has the updated schematic. Thnx!
